### DISCUSSION TITLE
How do i get the json of my theme.

### DISCUSSION THREAD

**Discussion by dancasmassonos** [JanDeDobbeleer/oh-my-posh#4980] [General]
I have an interesting problem. In PS profile of vscode I in error put
oh-my-posh init pwsh --config "$env:POSH_THEMES_PATH\M365Princess" | Invoke-Expression
something rather nice loaded. It has some problems that I was going to tweak but the theme is for sure not. Princess.

How do i find it? 
PS I looked through every theme.

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/91165638/98d70bd3-464e-4f57-af61-6bfc5a8a10a7)

**Comment by JanDeDobbeleer** [comment]
`oh-my-posh init pwsh --config "$env:POSH_THEMES_PATH\M365Princess.omp.json" | Invoke-Expression`

**Comment by dancasmassonos** [comment]
You are sayin the theme is Princess ?

**Reply by JanDeDobbeleer** [reply]
No, you didn't add `.omp.json` to the file name.
